src/main/java/seedu/souschef/logic/commands/DeleteCommand.java

```diff
@@ -30,7 +37,19 @@ public DeleteCommand(Model model, T toDelete) {
     public CommandResult execute(CommandHistory history) {
         model.delete(toDelete);
         model.commitAppContent();
-        return new CommandResult(String.format(MESSAGE_DELETE_RECIPE_SUCCESS, toDelete));
+
```

Comment:
Same here. Please remove all the related implementations! Context checking is redundant in a sense that we are already assigned specific context before reaching this class.